rosidl_typesupport_connext_c/cmake/rosidl_typesupport_connext_c_generate_interfaces.cmake

```diff
@@ -267,3 +257,30 @@ if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
 
   ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${Connext_LIBRARIES})
 endif()
+
+if(NOT "${_generated_msg_files}${_generated_srv_files} " STREQUAL " ")
+  find_package(ament_cmake_cppcheck)
```

Comment:
The check if testing is enabled wouldn't be sufficient. The linyers could be found with `QUIET` to suppress the warning in case the linters are not available.